tools/cellprofiler/cellprofiler.xml

```diff
@@ -0,0 +1,75 @@
+<tool id="cp319" name="Cellprofiler" version="19.05">
```

Comment:
the version should be the cellprofiler version imho, and therefor could also be  macro

Reply:
changed